src/OpenLoco/src/Map/Track/TrackData.cpp

```diff
@@ -1287,4 +1287,24 @@ namespace OpenLoco::World::TrackData
         assert(index < kRoadOccupationMasks.size());
         return kRoadOccupationMasks[index];
     }
+
+    // 0x004F865C
+    static constexpr std::array<uint8_t, 88> kUnk4F865C = { {
```

Comment:
Not sure what to name this. Its only user appears to be `VehicleHead::updateJunctionSignalLights`.

Reply:
Its an indicator as to the sharpness of the curve I'm not sure what you call that. Perhaps curvatureDegree